wire-ios-data-model/Source/Public/ZMMessage.h

```diff
@@ -60,15 +60,25 @@
 
 #pragma mark - ZMSystemMessageData
 
+/*
+ * Unfortunatly we can not remove deprecated cases of `ZMSystemMessageType` easily.
+ *
+ * The reason is that the values are persisted and stored and loaded later when the opens a conversation.
+ * To remove the cases we could have different strategies:
+ * 1. Assign the raw value of `int16_t` to each case, so that loading from store can map the correct values.
+ *    a. Then the mapping must fail gracefully for those values that can not be mapped then anymore.
+ *    b. Alternatively with a database migration all invalid values are deleted from the persisted store.
+ * 2. Keep the deprecated values in code.
+ */
```

Comment:
@netbe accidentally I merged everything in this PR, please look at the comment about deprecation here. Thank you :)